src/DI/ImageStorageExtension.php

```diff
@@ -10,40 +10,49 @@ class ImageStorageExtension extends CompilerExtension
 
 	/** @var mixed[] */
 	private $defaults = [
-		'data_path'          => '%wwwDir%/../public/data',
-		'data_dir'           => 'data',
-		'algorithm_file'     => 'sha1_file',
-		'algorithm_content'  => 'sha1',
-		'quality'            => 85,
-		'default_transform'  => 'fit',
+		'data_path' => '%wwwDir%/../public/data',
+		'data_dir' => 'data',
+		'algorithm_file' => 'sha1_file',
+		'algorithm_content' => 'sha1',
+		'quality' => 85,
+		'default_transform' => 'fit',
 		'noimage_identifier' => 'noimage/03/no-image.png',
-		'friendly_url'       => false,
+		'friendly_url' => false,
 	];
 
-	public function loadConfiguration(): void
-	{
-		$config = $this->_getConfig();
+	public function getConfigSchema (): Nette\Schema\Schema {
+		return Nette\Schema\Expect::structure([
+			'data_path' => Nette\Schema\Expect::string(),
```

Comment:
`string()->required()`

Reply:
Or get `wwwDir` from `$this->getContainerBuilder()->parameters['wwwDir']` and add `parameters['wwwDir'] . '/../public/data'` as default value. I would prefer required.